<?xml version="1.0" encoding="UTF-8"?>
<testng-results total="13" passed="0" failed="0" skipped="13">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-06-04T13:08:35Z" name="hbAxaTest" finished-at="2024-06-04T13:08:48Z" duration-ms="12755">
    <groups>
    </groups>
    <test started-at="2024-06-04T13:08:35Z" name="TC_PA_Life" finished-at="2024-06-04T13:08:45Z" duration-ms="10505">
      <class name="com.hbaxa.testcases.TC_PATest_001">
        <test-method is-config="true" signature="setup(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.hbaxa.testcases.TC_PATest_001@52815fa3]" started-at="2024-06-04T18:38:35Z" name="setup" finished-at="2024-06-04T18:38:35Z" duration-ms="0" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[10]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[TC_PA_Life_]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[null value in entry: url=null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: null value in entry: url=null
	at com.google.common.collect.CollectPreconditions.checkEntryNotNull(CollectPreconditions.java:32)
	at com.google.common.collect.SingletonImmutableBiMap.<init>(SingletonImmutableBiMap.java:42)
	at com.google.common.collect.ImmutableBiMap.of(ImmutableBiMap.java:72)
	at com.google.common.collect.ImmutableMap.of(ImmutableMap.java:124)
	at org.openqa.selenium.remote.RemoteWebDriver.get(RemoteWebDriver.java:273)
	at com.hbaxa.testcases.BaseClass.setup(BaseClass.java:141)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:577)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:514)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:215)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:178)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="paTestManual(java.lang.String)[pri:1, instance:com.hbaxa.testcases.TC_PATest_001@52815fa3]" started-at="2024-06-04T18:38:45Z" name="paTestManual" finished-at="2024-06-04T18:38:45Z" duration-ms="2" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[10]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- paTestManual -->
        <test-method signature="palistSearch(java.lang.String)[pri:2, instance:com.hbaxa.testcases.TC_PATest_001@52815fa3]" started-at="2024-06-04T18:38:45Z" name="palistSearch" finished-at="2024-06-04T18:38:45Z" depends-on-methods="com.hbaxa.testcases.TC_PATest_001.paTestManual" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TC_PATest_001.palistSearch(java.lang.String)[pri:2, instance:com.hbaxa.testcases.TC_PATest_001@52815fa3] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TC_PATest_001.palistSearch(java.lang.String)[pri:2, instance:com.hbaxa.testcases.TC_PATest_001@52815fa3] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1037)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- palistSearch -->
        <test-method signature="indexPASearch(java.lang.String)[pri:4, instance:com.hbaxa.testcases.TC_PATest_001@52815fa3]" started-at="2024-06-04T18:38:45Z" name="indexPASearch" finished-at="2024-06-04T18:38:45Z" depends-on-methods="com.hbaxa.testcases.TC_PATest_001.paTestManual" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TC_PATest_001.indexPASearch(java.lang.String)[pri:4, instance:com.hbaxa.testcases.TC_PATest_001@52815fa3] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TC_PATest_001.indexPASearch(java.lang.String)[pri:4, instance:com.hbaxa.testcases.TC_PATest_001@52815fa3] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1037)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- indexPASearch -->
        <test-method signature="paActivityLog(java.lang.String)[pri:6, instance:com.hbaxa.testcases.TC_PATest_001@52815fa3]" started-at="2024-06-04T18:38:45Z" name="paActivityLog" finished-at="2024-06-04T18:38:45Z" depends-on-methods="com.hbaxa.testcases.TC_PATest_001.paTestManual" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TC_PATest_001.paActivityLog(java.lang.String)[pri:6, instance:com.hbaxa.testcases.TC_PATest_001@52815fa3] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TC_PATest_001.paActivityLog(java.lang.String)[pri:6, instance:com.hbaxa.testcases.TC_PATest_001@52815fa3] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1037)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- paActivityLog -->
        <test-method signature="paGenerateLetter(java.lang.String)[pri:3, instance:com.hbaxa.testcases.TC_PATest_001@52815fa3]" started-at="2024-06-04T18:38:45Z" name="paGenerateLetter" finished-at="2024-06-04T18:38:45Z" depends-on-methods="com.hbaxa.testcases.TC_PATest_001.palistSearch" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TC_PATest_001.paGenerateLetter(java.lang.String)[pri:3, instance:com.hbaxa.testcases.TC_PATest_001@52815fa3] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TC_PATest_001.paGenerateLetter(java.lang.String)[pri:3, instance:com.hbaxa.testcases.TC_PATest_001@52815fa3] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1037)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- paGenerateLetter -->
        <test-method signature="paViewpage(java.lang.String)[pri:7, instance:com.hbaxa.testcases.TC_PATest_001@52815fa3]" started-at="2024-06-04T18:38:45Z" name="paViewpage" finished-at="2024-06-04T18:38:45Z" depends-on-methods="com.hbaxa.testcases.TC_PATest_001.paGenerateLetter" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TC_PATest_001.paViewpage(java.lang.String)[pri:7, instance:com.hbaxa.testcases.TC_PATest_001@52815fa3] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TC_PATest_001.paViewpage(java.lang.String)[pri:7, instance:com.hbaxa.testcases.TC_PATest_001@52815fa3] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1037)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- paViewpage -->
        <test-method is-config="true" signature="quitBrowser()[pri:0, instance:com.hbaxa.testcases.TC_PATest_001@52815fa3]" started-at="2024-06-04T18:38:45Z" name="quitBrowser" finished-at="2024-06-04T18:38:45Z" duration-ms="0" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitBrowser -->
      </class> <!-- com.hbaxa.testcases.TC_PATest_001 -->
      <class name="com.hbaxa.testcases.TC_PATestAdjustment_001">
        <test-method is-config="true" signature="setup(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.hbaxa.testcases.TC_PATestAdjustment_001@1cb346ea]" started-at="2024-06-04T18:38:45Z" name="setup" finished-at="2024-06-04T18:38:45Z" duration-ms="0" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="paTestAdjustment(java.lang.String)[pri:1, instance:com.hbaxa.testcases.TC_PATestAdjustment_001@1cb346ea]" started-at="2024-06-04T18:38:45Z" name="paTestAdjustment" finished-at="2024-06-04T18:38:45Z" duration-ms="8" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[10]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- paTestAdjustment -->
        <test-method signature="palistSearchAdjustment(java.lang.String)[pri:2, instance:com.hbaxa.testcases.TC_PATestAdjustment_001@1cb346ea]" started-at="2024-06-04T18:38:45Z" name="palistSearchAdjustment" finished-at="2024-06-04T18:38:45Z" depends-on-methods="com.hbaxa.testcases.TC_PATestAdjustment_001.paTestAdjustment" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TC_PATestAdjustment_001.palistSearchAdjustment(java.lang.String)[pri:2, instance:com.hbaxa.testcases.TC_PATestAdjustment_001@1cb346ea] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TC_PATestAdjustment_001.palistSearchAdjustment(java.lang.String)[pri:2, instance:com.hbaxa.testcases.TC_PATestAdjustment_001@1cb346ea] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1037)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- palistSearchAdjustment -->
        <test-method signature="palistAdjustmentLogLetter(java.lang.String)[pri:3, instance:com.hbaxa.testcases.TC_PATestAdjustment_001@1cb346ea]" started-at="2024-06-04T18:38:45Z" name="palistAdjustmentLogLetter" finished-at="2024-06-04T18:38:45Z" depends-on-methods="com.hbaxa.testcases.TC_PATestAdjustment_001.palistSearchAdjustment" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TC_PATestAdjustment_001.palistAdjustmentLogLetter(java.lang.String)[pri:3, instance:com.hbaxa.testcases.TC_PATestAdjustment_001@1cb346ea] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TC_PATestAdjustment_001.palistAdjustmentLogLetter(java.lang.String)[pri:3, instance:com.hbaxa.testcases.TC_PATestAdjustment_001@1cb346ea] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1037)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- palistAdjustmentLogLetter -->
        <test-method is-config="true" signature="quitBrowser()[pri:0, instance:com.hbaxa.testcases.TC_PATestAdjustment_001@1cb346ea]" started-at="2024-06-04T18:38:45Z" name="quitBrowser" finished-at="2024-06-04T18:38:45Z" duration-ms="0" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitBrowser -->
      </class> <!-- com.hbaxa.testcases.TC_PATestAdjustment_001 -->
    </test> <!-- TC_PA_Life -->
    <test started-at="2024-06-04T13:08:46Z" name="TC_Cashless_Life" finished-at="2024-06-04T13:08:46Z" duration-ms="8">
    </test> <!-- TC_Cashless_Life -->
    <test started-at="2024-06-04T13:08:46Z" name="TC_Reimbursment_Life" finished-at="2024-06-04T13:08:47Z" duration-ms="1213">
      <class name="com.hbaxa.testcases.TC_ReimDCTestManual_001">
        <test-method is-config="true" signature="setup(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.hbaxa.testcases.TC_ReimDCTestManual_001@45ac5f9b]" started-at="2024-06-04T18:38:46Z" name="setup" finished-at="2024-06-04T18:38:46Z" duration-ms="0" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[11]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[TC_Reimbursment_Life_]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[null value in entry: url=null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: null value in entry: url=null
	at com.google.common.collect.CollectPreconditions.checkEntryNotNull(CollectPreconditions.java:32)
	at com.google.common.collect.SingletonImmutableBiMap.<init>(SingletonImmutableBiMap.java:42)
	at com.google.common.collect.ImmutableBiMap.of(ImmutableBiMap.java:72)
	at com.google.common.collect.ImmutableMap.of(ImmutableMap.java:124)
	at org.openqa.selenium.remote.RemoteWebDriver.get(RemoteWebDriver.java:273)
	at com.hbaxa.testcases.BaseClass.setup(BaseClass.java:141)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:577)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:514)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:215)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:178)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="loginLevel0Test(java.lang.String)[pri:1, instance:com.hbaxa.testcases.TC_ReimDCTestManual_001@45ac5f9b]" started-at="2024-06-04T18:38:47Z" name="loginLevel0Test" finished-at="2024-06-04T18:38:47Z" duration-ms="1" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[11]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginLevel0Test -->
        <test-method signature="reimDCaptureManual(java.lang.String)[pri:2, instance:com.hbaxa.testcases.TC_ReimDCTestManual_001@45ac5f9b]" started-at="2024-06-04T18:38:47Z" name="reimDCaptureManual" finished-at="2024-06-04T18:38:47Z" depends-on-methods="com.hbaxa.testcases.TC_ReimDCTestManual_001.loginLevel0Test" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TC_ReimDCTestManual_001.reimDCaptureManual(java.lang.String)[pri:2, instance:com.hbaxa.testcases.TC_ReimDCTestManual_001@45ac5f9b] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TC_ReimDCTestManual_001.reimDCaptureManual(java.lang.String)[pri:2, instance:com.hbaxa.testcases.TC_ReimDCTestManual_001@45ac5f9b] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1037)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- reimDCaptureManual -->
        <test-method signature="reimDCaptureSearchClaim(java.lang.String)[pri:3, instance:com.hbaxa.testcases.TC_ReimDCTestManual_001@45ac5f9b]" started-at="2024-06-04T18:38:47Z" name="reimDCaptureSearchClaim" finished-at="2024-06-04T18:38:47Z" depends-on-methods="com.hbaxa.testcases.TC_ReimDCTestManual_001.reimDCaptureManual" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TC_ReimDCTestManual_001.reimDCaptureSearchClaim(java.lang.String)[pri:3, instance:com.hbaxa.testcases.TC_ReimDCTestManual_001@45ac5f9b] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TC_ReimDCTestManual_001.reimDCaptureSearchClaim(java.lang.String)[pri:3, instance:com.hbaxa.testcases.TC_ReimDCTestManual_001@45ac5f9b] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1037)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- reimDCaptureSearchClaim -->
        <test-method signature="logoutLevel0Test(java.lang.String)[pri:4, instance:com.hbaxa.testcases.TC_ReimDCTestManual_001@45ac5f9b]" started-at="2024-06-04T18:38:47Z" name="logoutLevel0Test" finished-at="2024-06-04T18:38:47Z" depends-on-methods="com.hbaxa.testcases.TC_ReimDCTestManual_001.reimDCaptureManual" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TC_ReimDCTestManual_001.logoutLevel0Test(java.lang.String)[pri:4, instance:com.hbaxa.testcases.TC_ReimDCTestManual_001@45ac5f9b] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TC_ReimDCTestManual_001.logoutLevel0Test(java.lang.String)[pri:4, instance:com.hbaxa.testcases.TC_ReimDCTestManual_001@45ac5f9b] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1037)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logoutLevel0Test -->
        <test-method is-config="true" signature="quitBrowser()[pri:0, instance:com.hbaxa.testcases.TC_ReimDCTestManual_001@45ac5f9b]" started-at="2024-06-04T18:38:47Z" name="quitBrowser" finished-at="2024-06-04T18:38:47Z" duration-ms="0" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitBrowser -->
      </class> <!-- com.hbaxa.testcases.TC_ReimDCTestManual_001 -->
    </test> <!-- TC_Reimbursment_Life -->
    <test started-at="2024-06-04T13:08:48Z" name="TC_PA_DocID_Life" finished-at="2024-06-04T13:08:48Z" duration-ms="5">
    </test> <!-- TC_PA_DocID_Life -->
    <test started-at="2024-06-04T13:08:48Z" name="TC_NM_Life" finished-at="2024-06-04T13:08:48Z" duration-ms="10">
    </test> <!-- TC_NM_Life -->
  </suite> <!-- hbAxaTest -->
</testng-results>
